createBenchmark.sh

```diff
@@ -15,19 +15,32 @@ set -exo pipefail
 # Contains OS specific sed function
 source benchmarks/setup/utils.sh
 
-if [ -z $1 ]
-then
-  echo "Please provide a benchmark name! Format should be 'YOUR_NAME-TOPIC'"
+function printUsageAndExit {
+  printf "\nUsage ./createBenchmark <benchmark-name>\n"
   exit 1
+}
+
+if [[ -z $1 ]]
+then
+  echo "<benchmark-name> not provided"
+  printUsageAndExit
 fi
 benchmark=$1
 
+if [[ ! $benchmark =~ ^[a-z0-9][-a-z0-9]{0,61}[a-z0-9]$ ]]; then
```

Comment:
Specific reason why not using `^[a-z0-9]([-a-z0-9]{,61}[a-z0-9])?$`? Single letters are normally allowed.

Reply:
It forced a - to be present for anything longer than 1 character

Reply:
But that's interesting. We can rework it to also allow 1 char

Reply:
> It forced a - to be present for anything longer than 1 character

Not sure what you mean but above you can see in my tests https://github.com/camunda-cloud/zeebe/pull/8455#issuecomment-998507563 it worked also for `22`

Reply:
Hmm perhaps I had my test wrong. I'll have another look tomorrow :D

Reply:
I checked among many others `ab` is not accepted by `^[a-z0-9]([-a-z0-9]{,61}[a-z0-9])?$` see https://regex101.com/r/vjsrEy/1

We can use `^[a-z0-9]([-a-z0-9]{0,61}[a-z0-9])?$` see https://regex101.com/r/vjsrEy/2 🎉